ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -46,22 +53,38 @@ public async Task<IActionResult> OnGetAsync(int? pageIndex)
 
             var draftStatusList = new List<NotificationStatus>() {NotificationStatus.Draft};
             var nonDraftStatusList = new List<NotificationStatus>() {NotificationStatus.Notified, NotificationStatus.Denotified};
-            IQueryable<Notification> draftsQueryable = service.GetBaseQueryableNotificationByStatus(draftStatusList);
-            IQueryable<Notification> nonDraftsQueryable = service.GetBaseQueryableNotificationByStatus(nonDraftStatusList);
+            IQueryable<Notification> draftsQueryable = notificationService.GetBaseQueryableNotificationByStatus(draftStatusList);
+            IQueryable<Notification> nonDraftsQueryable = notificationService.GetBaseQueryableNotificationByStatus(nonDraftStatusList);
 
-            if (!String.IsNullOrEmpty(SearchParameters.IdFilter))
+            if (!string.IsNullOrEmpty(SearchParameters.IdFilter))
             {
                 SearchParamsExist = true;
-                draftsQueryable = service.FilterById(draftsQueryable, SearchParameters.IdFilter);
-                nonDraftsQueryable = service.FilterById(nonDraftsQueryable, SearchParameters.IdFilter);
+                draftsQueryable = searchService.FilterById(draftsQueryable, SearchParameters.IdFilter);
+                nonDraftsQueryable = searchService.FilterById(nonDraftsQueryable, SearchParameters.IdFilter);
             }
 
-            IQueryable<Notification> notificationIdsQueryable = service.OrderQueryableByNotificationDate(draftsQueryable)
-                                                                .Union(service.OrderQueryableByNotificationDate(nonDraftsQueryable));
+            if (SearchParameters.SexId != null)
+            {
+                SearchParamsExist = true;
+                draftsQueryable = searchService.FilterBySex(draftsQueryable, (int) SearchParameters.SexId);
+                nonDraftsQueryable = searchService.FilterBySex(nonDraftsQueryable, (int) SearchParameters.SexId);
+            }
+
+            if (SearchParameters.PartialDob != null && SearchParameters.PartialDob.Year != null)
+            {
+                SearchParamsExist = true;
+                SearchParameters.PartialDob.TryConvertToDateTimeRange(out DateTime? start, out DateTime? end);
+
+                draftsQueryable = searchService.FilterByPartialDate(draftsQueryable, SearchParameters.PartialDob);

```

Comment:
I think this wanted to use `start` and `end` - otherwise, why are we converting just above?